src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -262,20 +282,38 @@ const DomainFragment = ({
     });
     const data = isFirstTimeOpen
       ? [
-          ...followingDomains.map((item, index) => ({
-            ...item,
-            user_id_follower: item.user_id_follower
-          })),
           {separator: true},
-          ...unfollowingDomains.map((item, index) => ({
+          ...unfollowingDomains.map((item) => ({
             ...item,
             user_id_follower: item.user_id_follower
           }))
         ]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ternary operator is used to decide which data to use based on the `isFirstTimeOpen` flag. This is a good way to handle different scenarios in your application.
```javascript
const data = isFirstTimeOpen
  ? [
    {separator: true},
    ...unfollowingDomains.map((item) => ({
      ...item,
      user_id_follower: item.user_id_follower
    }))
  ]
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->